api/src/main/resources/Reference_Application_Numeric_Concepts-26-2.x.xml

```diff
@@ -8,6 +8,6 @@
 <concept_numeric concept_id="5089" hi_absolute="250.0" low_absolute="0.0" units="kg" allow_decimal="true"/>
 <concept_numeric concept_id="5090" hi_absolute="272.0" low_absolute="10.0" units="cm" allow_decimal="true"/>
 <concept_numeric concept_id="5092" hi_absolute="100.0" low_absolute="0.0" units="%" allow_decimal="false"/>
-<concept_numeric concept_id="5242" hi_absolute="999.0" hi_critical="120.0" hi_normal="70.0" low_absolute="0.0" units="" allow_decimal="false"/>
```

Comment:
Hello @HerbertYiga , Were you not supposed to add a new xml for Reference_Application_Numeric_Concepts-27-2.xml.

Reply:
@sherrif10  should i leave whatever is in Reference_Application_Numeric_Concepts-26-2.x.xml as it is?

Reply:
@HerbertYiga what do the wiki page instructions say?

Reply:
@dkayiwa  it says , copy the contents of the generated files to their respective xml files under https://github.com/openmrs/openmrs-module-referencemetadata/blob/master/api/src/main/resources  and its what i did for this pr